src/plugins/intel_cpu/tests/unit/shape_inference_test/squeeze_shape_inference_test.cpp

```diff
@@ -82,7 +99,10 @@ INSTANTIATE_TEST_SUITE_P(
            make_tuple(ShapeVector{{10, 1, 0, 1, 3, 1, 1}, {4}},
                       std::vector<int64_t>{1, -1, 3, -2},
                       StaticShape({10, 0, 3})),
-           make_tuple(ShapeVector{{10, 1, 0, 1, 3, 1, 1}, {}}, std::vector<int64_t>{}, StaticShape({10, 0, 3})),
+           make_tuple(ShapeVector{{10, 1, 0, 1, 3, 1, 1}}, std::vector<int64_t>{}, StaticShape({10, 0, 3})),
+           // don't know if this test case should pass
+           // ref https://docs.openvino.ai/latest/openvino_docs_ops_shape_Squeeze_1.html#doxid-openvino-docs-ops-shape-squeeze-1
+           // make_tuple(ShapeVector{{10, 1, 0, 1, 3, 1, 1}, {}}, std::vector<int64_t>{}, StaticShape({10, 0, 3})),
```

Comment:
@maxnick  please help to look at this test case.  whatever pattern is null or {}, the test case  can pass for ngraph shape infer. but for CPU custom shape infer, it only pass when pattern is null.  after check [doc](https://docs.openvino.ai/latest/openvino_docs_ops_shape_Squeeze_1.html#doxid-openvino-docs-ops-shape-squeeze-1), I still can't be sure about which is right.

Reply:
Why the test with scalar second input `{}` fails with the CPU specific implementation? What is the cause of the error?

Reply:
I  fix this issue in the custom squeeze shape infer code.